packages/kbn-pm/src/utils/bazel/install_tools.ts

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * and the Server Side Public License, v 1; you may not use this file except in
+ * compliance with, at your election, the Elastic License or the Server Side
+ * Public License, v 1.
+ */
+
+import { resolve } from 'path';
+import { spawn } from '../child_process';
+import { readFile } from '../fs';
+import { log } from '../log';
+
+export interface BazelToolsVersion {
+  bazelisk: string;
+  bazel: string;
+}
+
+async function readBazelToolsVersionFile(repoRootPath: string, versionFilename: string) {
+  const version = (await readFile(resolve(repoRootPath, versionFilename)))
+    .toString()
+    .split('\n')[0];
+
+  if (!version) {
+    throw new Error(`${versionFilename} file do not contain any version set`);
+  }
+
+  return version;
+}
+
+async function readBazelToolsVersion(repoRootPath: string) {
+  try {
+    const bazeliskVersion = await readBazelToolsVersionFile(repoRootPath, '.bazeliskversion');
+    const bazelVersion = await readBazelToolsVersionFile(repoRootPath, '.bazelversion');
+
+    return {
+      bazelisk: bazeliskVersion,
+      bazel: bazelVersion,
+    } as BazelToolsVersion;
+  } catch (e) {
+    throw new Error(`[bazel_tools] Failed on reading bazel tools versions\n ${e}`);
+  }
+}
+
+export async function installBazelTools(repoRootPath: string) {
+  log.debug(`[bazel_tools] reading bazel tools versions from version files`);
+  const bazelToolsVersions = await readBazelToolsVersion(repoRootPath);
```

Comment:
Mind if we remove this abstraction and just define the variables like we are on 33 and 34?

Reply:
yeah for sure, done!